CHANGELOG/CHANGELOG-1.8.0.md

```diff
@@ -40,6 +40,8 @@ It is advised not to have any pending pods or PVCs once the volume limit per nod
 ### Deprecation 
 
 - The Dell CSI Operator is no longer actively maintained or supported. It will be deprecated in CSM 1.9. It is highly recommended that you use [CSM Operator](https://dell.github.io/csm-docs/docs/deployment/csmoperator/) going forward.
+- CSM for PowerMax linked Proxy mode for [CSI reverse proxy is no longer actively maintained or supported](https://dell.github.io/csm-docs/csm-docs/docs/csidriver/release/powermax/). It will be deprecated in CSM 1.9. It is highly recommended that you use stand alone mode going forward.
+- The CSM Authorization RPM is no longer actively maintained or supported. It will be deprecated in CSM 1.10.0. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward.
```

Comment:
Can this be changed to "The CSM Authorization RPM will be deprecated in a future release. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward."?

Reply:
Updated